docs/developers/tutorial.md

```diff
@@ -747,13 +747,11 @@ custom repository like Bintray or a private Nexus.
 
 ```scala
 // build.sbt
-import com.geirsson.coursiersmall.{Repository => R}
+import coursierapi.{MavenRepository => MavenR, IvyRepository => IvyR}
```

Comment:
Is there a reason we can't import `MavenRepository` directly? Is there a conflict with sbt APIs? I personally try to avoid rename imports, I'm not sure why the original document recommended `Repository => R` 

I think it's clearer to avoid the import and use the fully qualified name directly
```
  coursierapi.MavenRepository.of("https://oss.sonatype.org/content/repositories/snapshots"),
  coursierapi.IvyRepository.of("https://dl.bintray.com/sbt/sbt-plugin-releases")
```

Reply:
Yes it's from [sbt](https://github.com/sbt/sbt/blob/576f738de7effd539bcc38cb90647c1c4096f060/sbt/src/main/scala/sbt/Import.scala#L309), brought in scope by the implicit `sbt._`  in `.sbt`sources. I had done a rename to mimic what's currently on master, but I agree it's confusing. I'll update to use the fully qualified name.